README.md

```diff
@@ -0,0 +1,19 @@
+# chef
+Task for Epam Java Collection course
+
+In this project you can find salad making functionality. The program contains list of ingredients and two ready salads - Chef's and Russian salads.
+
+To launch the project using ide:
+Open the project and run main method in Main class.
+
+To launch the project without ide:
+1. Be sure you have downloaded JDK
```

Comment:
It would be great to add command to check that you have downloaded JDK
`java -version`

Also it would be great to mention that they need JDK 8

Reply:
Added